e2e/nopolicy/nopolicy_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+//go:build e2e
+
+package nopolicy
+
+import (
+	"context"
+	"flag"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/edgelesssys/contrast/e2e/internal/contrasttest"
+	"github.com/edgelesssys/contrast/e2e/internal/kubeclient"
+	"github.com/edgelesssys/contrast/internal/kuberesource"
+	"github.com/edgelesssys/contrast/internal/manifest"
+	"github.com/edgelesssys/contrast/internal/platforms"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// Persistent pod identifier of StatefulSet Coordinator is used.
+	coordinatorPod = "coordinator-0"
+	coordinator    = "coordinator"
+)
+
+// Namespace the tests are executed in.
+var (
+	imageReplacementsFile, namespaceFile, platformStr string
+	skipUndeploy                                      bool
+)
+
+// TestNoPolicy runs e2e test on the example of coordinator pod of the OpenSSL deployment, to ensure that container without policy annotation (default policy) do not start.
+func TestNoPolicy(t *testing.T) {
```

Comment:
Does this need to be its own e2e test? Then we'd need to decide when to run it. On the other hand, the unique test logic in here seems to be a couple of lines - maybe we can just add it as a subtest to openssl?

Reply:
Yes i get the point, but don't we require all of the contrast steps to successfully finish prior to the openssl test logic. This conflicts with using no policy annotation, so i would need to adjust the openssl test ?

Reply:
Actually, now that you mention it I think the test should go into https://github.com/edgelesssys/contrast/blob/main/e2e/policy/policy_test.go.

This test does not depend on the Coordinator etc., scheduling a pod with a single container and the right runtime class should suffice. Could look similar to [this](https://github.com/edgelesssys/contrast/blob/e9964e61853fe5bc2f5226820ddb3f97de2fbce4/e2e/getdents/getdents_test.go#L46), but with only `ct.Init` and `ct.Apply`.